mapbox_tilesets/scripts/cli.py

```diff
@@ -529,6 +530,16 @@ def _upload_source(
     if replace:
         method = "put"
 
+    token_parts = mapbox_token.split(".")
+    if len(token_parts) > 1:
+        while len(token_parts[1]) % 4 != 0:
+            token_parts[1] = token_parts[1] + "="
+        body = json.loads(base64.b64decode(token_parts[1]))
+        if "u" in body and username != body["u"]:
```

Comment:
what if "u" is not in body?

Reply:
You're right, this should probably be an error too.